src/validation.cpp

```diff
@@ -116,7 +116,6 @@ bool fPruneMode = false;
 bool fRequireStandard = true;
 bool fCheckBlockIndex = false;
 bool fCheckpointsEnabled = DEFAULT_CHECKPOINTS_ENABLED;
-size_t nCoinCacheUsage = 5000 * 300;
```

Comment:
In commit "Add CChainState::ResizeCoinsCaches" (df0f720cfdcc1ad70d5ef2656039f008b3bc4bba)

Just confirming but looks like this 1.5MB default cache size was only used by test cases previously, which are now using 8MB or 1KB caches depending on the case.